static/js/dropdown_list_widget.js

```diff
@@ -215,6 +217,13 @@ export class DropdownListWidget {
 
         this.setup_dropdown_widget(this.data);
 
+        if (this.render_pre_dropdown_icon) {
+            const $icon_container = $(`#${CSS.escape(this.container_id)} .pre-dropdown-icon`);
+            if ($icon_container.length) {
+                $icon_container.replaceWith(this.render_pre_dropdown_icon());
```

Comment:
we could also use `render_stream_privacy` if we're okay with adding the hashtag, and update it from `update_compose_stream_icon`